src/jit/codegenxarch.cpp

```diff
@@ -773,8 +773,8 @@ void CodeGen::genCodeForBinary(GenTree* treeNode)
     GenTreePtr op1 = treeNode->gtGetOp1();
     GenTreePtr op2 = treeNode->gtGetOp2();
 
-    // Commutative operations can mark op1 as contained to generate "op reg, memop/immed"
-    if (op1->isContained())
+    // Commutative operations can mark op1 as contained or used from spill to generate "op reg, memop/immed"
```

Comment:
This comment is actually referring to upstream phases (`Lowering` and `LSRA`) since it is talking about how something can be marked. I will change "used from spill" to "reg optional".